build.gradle

```diff
@@ -2,16 +2,13 @@ buildscript {
     repositories {
         google()
         jcenter()
-
-// uncomment for early access / dev releases
-//        maven { url 'https://dl.bintray.com/kotlin/kotlin-dev' }
-//        maven { url 'https://dl.bintray.com/kotlin/kotlin-eap' }
     }
     dependencies {
         classpath "com.android.tools.build:gradle:$android_gradle_plugin_version"
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
         classpath "org.jetbrains.kotlin:kotlin-serialization:$kotlin_version"
-        classpath 'com.google.gms:google-services:4.3.4'
+        classpath "com.google.gms:google-services:4.3.4"
+        classpath "com.jfrog.bintray.gradle:gradle-bintray-plugin:1.8.5"
```

Comment:
@avdyushin This also need to be added in example (and then get the version number from ext)

Reply:
Good catch, I totally forgot about this extra dependencies